tsl/src/bgw_policy/retention_api.c

```diff
@@ -215,10 +185,11 @@ policy_retention_add(PG_FUNCTION_ARGS)
 		Assert(list_length(jobs) == 1);
 		BgwJob *existing = linitial(jobs);
 
-		if (retention_window_equal(partitioning_type,
-								   existing->fd.config,
-								   window_type,
-								   window_datum))
+		if (policy_config_check_ht_lag_equality(existing->fd.config,
```

Comment:
The lag comes from the terminology for the arguments. The decision was to call the args for all the policies (except refresh)  as : xxxx_lag and  args for the refresh policy (start_interval, end_interval) since it deals with an interval.